user-guide/Advanced_Modules/Dashboards_and_Low_Code_Apps/GQI/Extensions/API_Reference/GQIColumn.md

```diff
@@ -1,12 +1,31 @@
 ---
-uid: GQIColumn
+uid: GQI_GQIColumn
 ---
 
-# GQIColumn
+# GQIColumn Class
 
-The *GQIColumn* object is an abstract class, with the derived types *GQIStringColumn*, *GQIBooleanColumn*, *GQIIntColumn*, *GQIDateTimeColumn*, *GQIDoubleColumn*, and (from DataMiner 10.3.9/10.4.0 onwards<!-- RN 36717 -->) *GQITimeSpanColumn*, and with the following properties:
+## Definition
+
+Namespace: `Skyline.DataMiner.Analytics.GenericInterface`  
+Assembly: `SLAnalyticsTypes.dll`
+
+Provides the base class for a column in GQI.
+
+## Derived types
+
+The *GQIColumn* object is an abstract class, with the derived types:
```

Comment:
This sentence is probably redundant?
Also, might be best to order the derived types alphabetically in the list to find them quicker.